src/Compilers/VisualBasic/Portable/Binding/Binder_Statements.vb

```diff
@@ -65,7 +65,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
                 Case SyntaxKind.ElseIfStatement
                     ' ElseIf without a preceding If.
-                    Debug.Assert(node.ContainsDiagnostics)
+                    ' Debug.Assert(node.ContainsDiagnostics)
```

Comment:
What test failed here and with other assert below? Are there cases where we're producing a bound tree with `BoundBadStatement` but without diagnostics?